drivers/gpio/Kconfig.fxl6408

```diff
@@ -0,0 +1,21 @@
+# ST Microelectronics STM32L151XC MCU
```

Comment:
This comment about STM32L151XC seems out of place.

Reply:
I've removed this line